public/index.html

```diff
@@ -6,7 +6,7 @@
     <meta name="theme-color" content="#000000">
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json">
     <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
-    <title>Hathor Network - Admin</title>
+    <title>Hathor Network - Explorer</title>
```

Comment:
I would change to "Hathor Public Explorer", or just "Explorer - Hathor Network". What do you think?

Reply:
I agree, changed to "Explorer - Hathor Network"